constants/labels.en.js

```diff
@@ -96,5 +96,58 @@ const labels = {
     Maple: {
         labelimgmaple: 'mapleleave',
     },
+
+    form: {        
+        labels : {
```

Comment:
Please use an scalable format like and array of objects called fields. Each field would have the props like: label, placeholder, type and etc